cloud/roles/reserve_static_ip/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- name:   Check if static IP address already exists
+  shell: >
+      gcloud compute addresses list
+      --project {{ gke.project }}
+      | grep {{ name_of_ip }}
+  register:      ip_address_result
+  ignore_errors: yes
+
+- name:    Reserve static IP address for the "{{ server_name }}" instance
+  command: >
+      gcloud compute addresses create {{ name_of_ip }}
+      --description "Static IP attached to {{ server_name }} instance"
+      --project     {{ gke.project }}
+      --region      europe-west3
```

Comment:
Zone should be taken from `concent-deployment-values`.

Reply:
This is not a zone. I will add the `region` variable to the `concent-deployment-values` ​​repository.

Reply:
Right. Anyway, point still stands - it should not be hard-coded here.